src/NETCDF/dump_netcdf_mpiio.h

```diff
@@ -40,12 +40,26 @@ class DumpNetCDFMPIIO : public DumpCustom {
   virtual void write();
 
  private:
+   // type of quantity for per-atom values (used to get the unit)
+   enum Quantity {
```

Comment:
We try to avoid redundant definitions of constants. So it would be better to have this in a separate header `netcdf_quantity.h` as `enum LAMMPS_NS::NetCDFQuantity` and, if possible, only use a forward declaration in the dump style headers (to reduce dependencies).